scripts/scheduled-task-commands/automatedEmailReminderVAPO.php

```diff
@@ -0,0 +1,34 @@
+<?php
+// this file will need to be added, Pete's destruction ticket has it already.
+require_once 'globals.php';
+require_once APP_PATH . '/Leaf/Db.php';
+require_once APP_PATH . '/Leaf/ErrorNotify.php';
+
+$startTime = microtime(true);
+
+$db = new App\Leaf\Db(DIRECTORY_HOST, DIRECTORY_USER, DIRECTORY_PASS, 'national_leaf_launchpad');
+$errorNotify = new App\Leaf\ErrorNotify();
+$siteList = $db->query("SELECT `site_path` FROM `sites` WHERE `site_type` = 'portal' AND `isVAPO` = 'true'");
+$dir = '/var/www/html';
+
+$failedArray = [];
+
+foreach ($siteList as $site) {
+    echo "Portal: " . $dir . $site['site_path'] . '/scripts/automated_email.php' . "\r\n";
```

Comment:
I'm thinking we should make the output (echos) actually go to a log file.  Make it easier to troubleshoot and we can get Dynatrace to watch it that way.